o1vm/src/interpreters/riscv32im/interpreter.rs

```diff
@@ -2193,7 +2193,17 @@ pub fn interpret_mtype<Env: InterpreterEnv>(env: &mut Env, instr: MInstruction)
             env.set_next_instruction_pointer(next_instruction_pointer + Env::constant(4u32));
         }
         MInstruction::Divu => {
-            unimplemented!("Divu")
```

Comment:
it would be nice to comment the implementation from https://msyksphinz-self.github.io/riscv-isadoc/html/rvm.html

Reply:
we can follow up with this documentation in a future pr